src/displayapp/screens/settings/Settings.cpp

```diff
@@ -36,10 +36,10 @@ bool Settings::OnTouchEvent(Pinetime::Applications::TouchEvents event) {
 std::unique_ptr<Screen> Settings::CreateScreen1() {
 
   std::array<Screens::List::Applications, 4> applications {{
-    {Symbols::sun, "Display", Apps::SettingDisplay},
-    {Symbols::eye, "Wake Up", Apps::SettingWakeUp},
-    {Symbols::clock, "Time format", Apps::SettingTimeFormat},
-    {Symbols::home, "Watch face", Apps::SettingWatchFace},
+    {Symbols::list, "About", Apps::SysInfo},
+    {Symbols::airplane, "Airplane mode", Apps::SettingAirplaneMode},
+    {Symbols::batteryHalf, "Battery", Apps::BatteryInfo},
+    {Symbols::sun, "Display", Apps::SettingDisplay}
```

Comment:
The order shouldn't be alphabetized and shouldn't be changed in this PR.

Reply:
I agree with @Riksu9000, I'm not sure the alphabetized order is the best one, and I would rather like to see this change in a dedicated PR.

Reply:
I disagree. While I concede this may not be an ordering that makes 100% of people happy (does that even exist?), I feel strongly that the aribtrary order is wrong from a UX standpoint because it creates an unecessary cognitive burden on the user.

If we order the options alphabetically now I expect it will kick off a serious, evidence based discussion on how this is best resolved. Folks posting about which option they use the most (FWIW I use "About" the most) is not moving the ball forward and just adding new options in random spots is procrastination.

At the very least, alphabetical ordering is a common, accepted UX pattern for options that is widely understood.

Reply:
I agree that alphabetical ordering has its benefits. However when changing languages should the order change according to the used language? Alphabetical ordering isn't common for settings on other platforms either. At the very least this will not be changed in a PR about airplane mode.